src/Http/Middleware/HandleErrors.php

```diff
@@ -52,7 +52,7 @@ public function __invoke($error, Request $request, Response $response, callable
             $status = $errorCode;
         }
 
-        if ($this->debug && $errorCode !== 404) {
+        if ($this->debug && ! in_array($errorCode, [403, 404])) {
```

Comment:
One more thing: If we decide not to merge this, we are bound to get more reports like this...